sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -330,7 +330,7 @@ class DataSourceV2Strategy(session: SparkSession) extends Strategy with Predicat
       SetCatalogAndNamespaceExec(catalogManager, catalogName, ns) :: Nil
 
     case r: ShowCurrentNamespace =>
-      ShowCurrentNamespaceExec(r.output, r.catalogManager) :: Nil
+      ShowCurrentNamespaceExec(r.output, r.catalogManager.get) :: Nil
```

Comment:
We don't need to pass the catalog manager from logical plan to physical plan, we can get it directly in the physical plan
```
val catalogManager = session.sessionState.catalogManager
...
```
see  https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/SparkPlan.scala#L60

Reply:
then the logical plan can be `case class ShowCurrentNamespace extends LeafCommand`